addons/medical/functions/fnc_addToLog.sqf

```diff
@@ -9,27 +9,26 @@
  * 3: The arguments for localization <ARRAY>
  *
  * Return Value:
- * nil
+ * None
  *
  * Public: Yes
  */
 
 #include "script_component.hpp"
 
-private ["_unit", "_type", "_message", "_arguments", "_lastNumber", "_moment", "_logVarName", "_log","_newLog", "_logs"];
-_unit = _this select 0;
-_type = _this select 1;
-_message = _this select 2;
-_arguments = _this select 3;
+private ["_moment", "_logVarName", "_log","_newLog", "_logs"];
+params ["_unit", "_type", "_message", "_arguments"];
 
 if (!local _unit) exitwith {
-    [_this, QUOTE(DFUNC(addToLog)), _unit] call EFUNC(common,execRemoteFnc); /* TODO Replace by event system */
+    [_this, QFUNC(addToLog), _unit] call EFUNC(common,execRemoteFnc); /* TODO Replace by event system */
 };
 
-_lastNumber = date select 4;
-_moment = format["%1:%2",date select 3, _lastNumber];
-if (_lastNumber < 10) then {
-    _moment = format["%1:0%2",date select 3, _lastNumber];
+date params ["", "", "", "_minute", "_hour"];
+
+_moment = if (_minute < 10) then {
```

Comment:
posible to use this?
`_moment = format [ (["%1:%2", "%1:0%2"] select (_minute < 10)), _hour, _minute]`